The “EditorTrackSegmentType  section of “editors.xsd  is missing a “decorations  element.

geowar1 opened the pull request.

pabender commented:
There is something very wrong with this PR.

It touches almost 5000 files, and reverts several changes that have been made over (at least) the last several weeks.

The problem is probably the second commit in the commit sequence, which reverts merging master into the branch.

Until you fix this problem, we can't merge this in.

dsand47 commented:
Paul,

George has requested that I submit the schema change for him.  I will be doing that tomorrow.  You can close this PR.

Dave Sand

geowar1 commented:
Yeah, git's lost its mind again. GitHub Desktop says I’ve changed one file, I push the PR and github says I’ve changed 4739. Gerrrrrrr. I’m closing this one. DaveS is submitting it for me. Now returning to my retirement (already in progress).
